### PR TITLE
Add limix-core

### PR DESCRIPTION
**Pull request by Horta**

### CODE REVIEW

File: recipes/limix-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  requires:
+    - pytest >=3
+  imports:
+    - limix_core
+  commands:
+    - python -c "import sys; import limix_core; sys.exit(limix_core.test())"
+
+about:
+  home: https://github.com/limix/limix-core
+  license: MIT
```
Comment (synapticarbors) [inline recipes/limix-core/meta.yaml]: It appears as if the license should be `APACHE-2.0`:

https://github.com/limix/limix-core/blob/master/LICENSE

Comment (Horta) [inline recipes/limix-core/meta.yaml]: Apache-2.0 or Apache 2.0? because I think I have used the second one before here...

Comment (synapticarbors) [inline recipes/limix-core/meta.yaml]: My understanding is that conda-forge prefers the abbreviations found on the following page:

https://opensource.org/licenses/alphabetical

**Comment by Horta** [comment]
Ready to merge =)

**Pull request merged**